### PR TITLE
incomplete pip command in warning

### PR DESCRIPTION
**Pull request by dmartin35**
check_newer_version logs full pip command with --pre option when latest version is a release candidate

### CODE REVIEW

File: chaostoolkit/check.py [hunk 1]
```diff
@@ -26,13 +26,14 @@ def check_newer_version(command: str):
             payload = r.json()
             latest_version = payload["version"]
             if payload.get("up_to_date") is False:
+                options = '--pre' if 'rc' in latest_version else ''
                 logger.warning(
                     "\nThere is a new version ({v}) of the chaostoolkit "
                     "available.\n"
                     "You may upgrade by typing:\n\n"
-                    "$ pip install -U chaostoolkit\n\n"
+                    "$ pip install -U chaostoolkit {opt}\n\n"
```
Comment (Lawouach) [inline chaostoolkit/check.py]: Could we put {opt} before `-U` ?

Comment (dmartin35) [reply chaostoolkit/check.py]: yes, sure! i'll make the change.

**Comment by dmartin35** [comment]
I also made a change to handle all options in a list (filtered for None avoiding multiple spaces between missing options)

File: chaostoolkit/check.py [hunk 2]
```diff
@@ -26,14 +26,16 @@ def check_newer_version(command: str):
             payload = r.json()
             latest_version = payload["version"]
             if payload.get("up_to_date") is False:
-                options = '--pre' if 'rc' in latest_version else ''
+                options = filter(None, [
```
Comment (Lawouach) [inline chaostoolkit/check.py]: While I appreciate one liners, I don't think that reads as simply as: 

```
options = '--pre -U' if 'rc' in latest_version else '-U'
```

or I may be missing something?

Comment (dmartin35) [reply chaostoolkit/check.py]: Nope you're right, code was intended to support multiple options (that can be added later on) but that made the code less clear/concise. 
i'll change it.

Comment (Lawouach) [reply chaostoolkit/check.py]: Thanks!

**Comment by Lawouach** [comment]
You may have to rebase your branch against master and force push again mate :)

**Comment by dmartin35** [comment]
argh... missed the DCO in my commit message, can't amend via the UI, i'll do it tonight at home

**Comment by Lawouach** [comment]
Don't worry about it. You can do that when you'll rebase and amend the commit :)

Thanks anywa mate!

**Pull request merged**